src/main.cpp

```diff
@@ -472,6 +472,10 @@ int main(int argc, char **argv)
 		#endif  // SIGABRT_COMPAT
 	#endif
 
+
+	#ifdef __unix__
+		std::signal(SIGPIPE, SIG_IGN);
```

Comment:
No. SIGINT fires the first time calling NonCtrl. We then initiate a shutdown, and change the handler to ignore future SIGINT signals.

WIth SIGPIPE, we want to ignore all SIGPIPE signals.